### PR TITLE
Guest Checkout sucess add register #4356

### PR DESCRIPTION
**Pull request by domko17**
## Issue Reference

#4356 
## Documentation
- [ ] My pull request requires an update on the documentation repository.

### CODE REVIEW

File: packages/Webkul/Customer/src/Http/Controllers/RegistrationController.php [hunk 1]
```diff
@@ -180,6 +180,7 @@ public function verifyAccount($token)
 
         if ($customer) {
             $customer->update(['is_verified' => 1, 'token' => 'NULL']);
+            Order::where('customer_email', $customer->email)->update(['customer_id' => $customer->id]);
```
Comment (devansh-webkul) [inline packages/Webkul/Customer/src/Http/Controllers/RegistrationController.php:183]: 1 line between 182 and 183.

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

File: packages/Webkul/Customer/src/Http/Controllers/RegistrationController.php (hunk 1 above)
Comment (domko17) [reply packages/Webkul/Customer/src/Http/Controllers/RegistrationController.php:183]: what? something is wrong ?

Comment (devansh-webkul) [reply packages/Webkul/Customer/src/Http/Controllers/RegistrationController.php:183]: Add line here.

Comment (domko17) [reply packages/Webkul/Customer/src/Http/Controllers/RegistrationController.php:184]: its done

**Review by ghermans — APPROVED** [review]

**Pull request merged**